model/DataStore/PersistDataService.php

```diff
@@ -164,4 +145,49 @@ private function getFileSystemManager(): FileSystemService
     {
         return $this->getServiceLocator()->get(FileSystemService::SERVICE_ID);
     }
+
+    private function addMetaDataFile($zipFile, string $fileNameToAdd, string $content): bool
```

Comment:
Is it possible to declare `$zipFile` type here (and [here](https://github.com/oat-sa/extension-tao-delivery-rdf/pull/376/files#diff-d7dda33bcb2b0511001c7d9b6fa95a73ce058a954084c4bf1db51ae715a22fb0R166))?

Reply:
done